### PR TITLE
simplify assent/consent title

### PR DESCRIPTION
**Pull request by lucasxsong**
## Types of changes

What types of changes does this pull request introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactoring (non-breaking change that does not add functionality but makes code cleaner or more efficient)
- [ ] Documentation Update
- [ ] Tests (new or updated tests)
- [ ] Style (changes to code styling)
- [ ] CI (continuous integration changes)
- [ ] Repository Maintenance
- [ ] Other (please describe below)

## Checklist

- [ ] I have read the [guidelines for contributing](https://github.com/yeatmanlab/roar-dashboard/blob/main/.github/CONTRIBUTING.md).
- [ ] The changes in this PR are as small as they can be. They represent one and only one fix or enhancement.
- [ ] Linting checks pass with my changes.
- [ ] Any existing unit tests pass with my changes.
- [ ] Any existing end-to-end tests pass with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] If this PR fixes an existing issue, I have added a unit or end-to-end test that will detect if this issue reoccurs.
- [ ] I have added JSDoc comments as appropriate.
- [ ] I have added the necessary documentation to the [roar-docs repository](https://github.com/yeatmanlab/roar-docs).
- [ ] I have shared this PR on the roar-pr-reviews channel (if I have access)
- [ ] I have linked relevant issues (if any)

### CODE REVIEW

**Review by Emily-ejag — APPROVED** [review]
LGTM

**Comment by Emily-ejag** [comment]
You will need to change the cypress test, to only look for Consent :)

**Review by Emily-ejag — DISMISSED** [review]

File: src/components/ConsentModal.vue [hunk 1]
```diff
@@ -53,12 +53,11 @@ onMounted(() => {
   dialogVisible.value = true;
 
   const acceptIcon = computed(() => (isSubmitting.value ? 'pi pi-spin pi-spinner mr-2' : 'pi pi-check mr-2'));
+  const header = props.consentType.includes('consent') ? `Consent` : `Assent`;
```
Comment (richford) [inline src/components/ConsentModal.vue:56]: It seems like you got rid of the translations in this refactor. Was that intentional? If so, can you justify the removal?

**Review by richford — CHANGES_REQUESTED** [review]

File: cypress/e2e/participant/default-tests/checkLegalDoc.cy.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ describe('Test to maintain that assent form shows in when signing in with an un-
     // how can we write some logic to reset the already played
     cy.login(test_login, test_pw);
     cy.visit('/');
-    cy.get('.p-dialog-title', { timeout: timeout }).contains('CONSENT').should('be.visible');
+    cy.get('.p-dialog-title', { timeout: timeout }).contains('Consent').should('be.visible');
```
Comment (richford) [inline cypress/e2e/participant/default-tests/checkLegalDoc.cy.js:15]:
```suggestion
    cy.get('.p-dialog-title', { timeout: timeout }).contains(i18n.t('consentModalo.consentTitle')).should('be.visible');
```

**Review by richford — CHANGES_REQUESTED** [review]

File: cypress/e2e/participant/default-tests/checkLegalDoc.cy.js (hunk 2 above)
Comment (lucasxsong) [reply cypress/e2e/participant/default-tests/checkLegalDoc.cy.js:15]: Thanks for catching this!

**Review by richford — DISMISSED** [review]

**Review by Emily-ejag — DISMISSED** [review]

**Review by Emily-ejag — APPROVED** [review]

**Pull request merged**